pkg/watcher/results/results.go

```diff
@@ -193,10 +193,14 @@ func (c *Client) ensureResult(ctx context.Context, o Object, opts ...grpc.CallOp
 
 // parseAnnotations attempts to return the provided value as a map of strings.
 func parseAnnotations(annotationKey, value string) (map[string]string, error) {
-	var annotations map[string]string
-	if err := json.Unmarshal([]byte(value), &annotations); err != nil {
+	var data map[string]interface{}
```

Comment:
There was a PR by @enarha before to replace all `interface{}` with `any`.
```suggestion
	var data map[string]any
```

Reply:
It's shorter and more convenient and I was striving for standardization. See 1ad89d22734c35ab7c2171cae26e4950de8841e9